### ISSUE TITLE
SmarterQueue

### ISSUE THREAD

**Issue by danielepolencic** [uasabi/noderosso#15]
TL;DR: Create a node that can release messages on a schedule.

The idea is to recreate a node that can (partially) replace [SmarterQueue.](https://smarterqueue.com/)

The node should accept five input:

1. a `TWEET.V1` message with a payload that includes the text and a collection of images as links.
2. a `PUBLISHED.V1` message with the id of the item that was just published.
3. a `TICK.V1` message to check for messages to be published.
4. a `FLUSH.V1` message to empty the state.
5. a `RESCHEDULE.V1` message to reschedule all the tweets against the current timings.

And should provide the following output:

1. a `PUBLISH_TWEET.V1` message with the id of the tweet, the message and any image as a collection of links.

In the editor, you should create a widget where you can list timings such as:

1. Wednesday at 2pm
2. Thursday at 3pm
3. Friday at 4pm

The editor should look like this:

<img width="526" alt="Screenshot 2021-02-05 at 22 27 45" src="https://user-images.githubusercontent.com/1130499/107046264-66918180-6801-11eb-98d9-3db8ba542609.png">

There's an example on the classifier node on how you can achieve that.

## Input

The input should be:

- `text: string`: the text of the tweet.
- `link: string | undefined`: the link for the tweet.
- `images: string[]`:

The tweet should be stored alongside the following properties:

- `publishedAt: {datetime: string, twitterId: string}[]`. The number of times the tweet was published.
- `nextSchedule: string | null`. the date for when the tweet is scheduled to be published next.
- `id: string`. UUID.

The `nextSchedule` property should be assigned the next available slot.

**Comment by danielepolencic** [comment]
Added
